ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -53,23 +54,34 @@ - (void)acquireToken:(NSString *)apiId
     [self ensureRequest];
     NSString* telemetryRequestId = [_requestParams telemetryRequestId];
     
-    __block NSString* log = [NSString stringWithFormat:@"##### BEGIN acquireToken%@ (authority = %@, resource = %@, clientId = %@, idtype = %@) #####",
-                             _silent ? @"Silent" : @"", _requestParams.authority, _requestParams.resource, _requestParams.clientId, [_requestParams.identifier typeAsString]];
+    AD_LOG_INFO(_requestParams.correlationId, @"##### BEGIN acquireToken %@ #####", _silent ? @"Silent" : @"");
+    
+    BOOL isKnownHost = [ADAuthorityUtils isKnownHost:_requestParams.authority];
+    AD_LOG(ADAL_LOG_LEVEL_INFO, _requestParams.correlationId, !isKnownHost, @"authority: %@", _requestParams.authority);
+    AD_LOG_INFO_PII(_requestParams.correlationId, @"resource = %@", _requestParams.resource);
+    AD_LOG_INFO_PII(_requestParams.correlationId, @"clientId = %@", _requestParams.clientId);
+    AD_LOG_INFO(_requestParams.correlationId, @"idtype = %@", [_requestParams.identifier typeAsString]);
     AD_LOG_INFO_PII(_requestParams.correlationId, @"userId = %@", _requestParams.identifier.userId);
     
     ADAuthenticationCallback wrappedCallback = ^void(ADAuthenticationResult* result)
     {
         if (result.status == AD_SUCCEEDED)
         {
-            AD_LOG_INFO(result.correlationId, @"##### END %@ succeeded. #####", log);
+            AD_LOG_INFO(result.correlationId, @"##### END succeeded. #####");
         }
         else
         {
             ADAuthenticationError* error = result.error;
-            AD_LOG_INFO(result.correlationId, @"##### END %@ failed { domain: %@ code: %ld protocolCode: %@ ", log, error.domain, (long)error.code, error.protocolCode);
+            AD_LOG_INFO(result.correlationId, @"##### END failed { domain: %@ code: %ld protocolCode: %@ ", error.domain, (long)error.code, error.protocolCode);
             AD_LOG_INFO_PII(result.correlationId, @"errorDetails: %@ ", error.errorDetails);
         }
         
+        AD_LOG(ADAL_LOG_LEVEL_INFO, _requestParams.correlationId, !isKnownHost, @"authority: %@", _requestParams.authority);
+        AD_LOG_INFO_PII(_requestParams.correlationId, @"resource = %@", _requestParams.resource);
+        AD_LOG_INFO_PII(_requestParams.correlationId, @"clientId = %@", _requestParams.clientId);
+        AD_LOG_INFO(_requestParams.correlationId, @"idtype = %@", [_requestParams.identifier typeAsString]);
+        AD_LOG_INFO_PII(_requestParams.correlationId, @"userId = %@", _requestParams.identifier.userId);
+        
```

Comment:
nit: can we possibly collapse this and previous log lines by e.g. using log string and logPII string? It's just making it a little bit difficult to read the code itself because there's so many logs.